lib/room.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../lib/reservation.rb'
+require 'awesome_print'
+require 'pry'
+
+module Hotel
+  class Room
+    attr_reader :room_number, :price, :reservations, :array
+    attr_accessor :reservation_info
+
+    @@array = []
```

Comment:
Not a good variable name - you should always use some contextual referencing the type of data